app/src/main/kotlin/com/intive/picover/main/navigation/launcher/Launcher.kt

```diff
@@ -0,0 +1,20 @@
+package com.intive.picover.main.navigation.launcher
+
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+@Singleton
+class Launcher @Inject constructor() {
```

Comment:
The idea is to inject that later into ios view controller, in the swift code, and launch there sign in screen from firebase auth UI for swift. In theory should work 😄  We should be able to use this https://skie.touchlab.co/features/flows to observe flow in swift

Reply:
Sounds promising 😄